front/src/modules/companies/components/FilterDropdownCompanySearchSelect.tsx

```diff
@@ -0,0 +1,43 @@
+import { filterDropdownSearchInputScopedState } from '@/filters-and-sorts/states/filterDropdownSearchInputScopedState';
+import { filterDropdownSelectedEntityIdScopedState } from '@/filters-and-sorts/states/filterDropdownSelectedEntityIdScopedState';
+import { useRecoilScopedState } from '@/recoil-scope/hooks/useRecoilScopedState';
+import { useRecoilScopedValue } from '@/recoil-scope/hooks/useRecoilScopedValue';
+import { useFilteredSearchEntityQuery } from '@/relation-picker/hooks/useFilteredSearchEntityQuery';
+import { Entity } from '@/relation-picker/types/EntityTypeForSelect';
+import { FilterDropdownEntitySearchSelect } from '@/ui/components/table/table-header/FilterDropdownEntitySearchSelect';
+import { TableContext } from '@/ui/tables/states/TableContext';
+import { getLogoUrlFromDomainName } from '@/utils/utils';
+import { useSearchCompanyQuery } from '~/generated/graphql';
+
+export function FilterDropdownCompanySearchSelect() {
+  const filterDropdownSearchInput = useRecoilScopedValue(
+    filterDropdownSearchInputScopedState,
+    TableContext,
```

Comment:
That's already a big improvement. I wonder what we aim for long term with table and filters.
What I currently imagine:
- table module (that can be used by People, Companies pages/module)
- kanban module (that can be used by Opportunities pages/module
- filter module (thant can be used by People, Companies, Opportunities pages/module
- People (resp. Companies / Opportunities) module is composing: with table, filters. so it's the one providing to this FilterDropdownCompanySearchSelect, this "TableContext" that we would light to rename to something more generic.

LGTM for this PR, just thinking what the next step would be